_targets.R

```diff
@@ -28,21 +46,36 @@ list(
   #   tar_target(find_oldest_sites(states, parameter)
   #   )
   # ),
-
-  tar_map(
-    values = tibble(state_abb = states) %>%
-      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
-    tar_target(nwis_inventory, filter(oldest_active_sites, state_cd == state_abb)),
-    tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
-    tar_target(tally, tally_site_obs(nwis_data)),
-    tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter)),
-    names = state_abb
-    # Insert step for plotting data here
-  ),
   # Map oldest sites
+  mapped_by_state_targets,
+  tar_combine(
+    name = obs_tallies,
+    mapped_by_state_targets$tally,
+    command = bind_rows(!!!.x)
```

Comment:
Ah so `combine_obs_tallies()` in the `2_process/src/tally_site_obs.R` file can actually be deleted because you are just using `bind_rows()` here directly 👍